packages/navigation/src/index.ts

```diff
@@ -1,6 +1,8 @@
 import './useAutoFocus';
 
 export * from './UIGridList';
+// TODO: move it
```

Comment:
Will you?)

Reply:
I actually don't know, what do you think, should I? Or it's good enough to keep it there for now? :)

Reply:
Maybe it's better to move both in some kind of Lists folder?